docker-assets/auth-api-service.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/bash
+
+export PATH=$PATH:/opt/rubies/ruby-2.3.1/bin
+cd /opt/rh/auth-api
```

Comment:
Don't think this should be `rh` probably `manageiq`

Reply:
Or just `/opt/auth-api` since it has nothing to do with `rh` or `manageiq`?

Reply:
It might be weird to have a top-level "project" called auth-api though?  it's part of manageiq, so I'd prefer manageiq if we namespace at all.  if red hat happens to use it, it's all under the covers anyway.